scripts/mkcloud

```diff
@@ -1102,6 +1102,7 @@ These steps are expanding to the following steps:
         -> $(expand_steps instonly)
 
 Steps:
+    setuphost:      configure the host to run VMs
```

Comment:
well, its rather:
`prepare the host for being an mkcloud host`
because its more than just being able to run VMs